src/main/java/mate/academy/springboot/datajpa/controller/ProductController.java

```diff
@@ -59,7 +59,7 @@ public ProductResponseDto update(@PathVariable Long id,
         return productMapper.mapToDto(productService.update(product));
     }
 
-    @GetMapping("/price")
+    @GetMapping("/by-price") // price between
```

Comment:
```suggestion
    @GetMapping("/by-price")
```
price between is just another way to name endpoint